standalone-node/docs/prepare_bootable_usb_windows.md

```diff
@@ -40,6 +45,33 @@ This document explains the procedure to create a bootable USB device for Standal
    Get-NetAdapter | Where-Object Name -Like "*WSL*" | Enable-NetAdapter
    ```
 
+### Install Ubuntu 22.04 in a Hyper-V Virtual Machine
+
+1. Download the .ISO image for Ubuntu 22.04.5 LTS (Jammy Jellyfish).
+2. Start Hyper-V Manager and select *Action-> New-> Virtual Machine*.
```

Comment:
For someone to use hyper-v they should also enable virtualization technology in BIOS... these steps should be captured.

Reply:
how about we mention this or provide a link to customer that they need to enable virtualization.